### PR TITLE
Adding commit of wildcards removal in ignore-revs

### LINKED ISSUE #16747: Add 'wildcard import' to `.git-blame-ignore-revs`
* Obtain hash from https://github.com/ankidroid/Anki-Android/pull/16742
* Add it to the file, along with a one-line comment

### PR DESCRIPTION
**Pull request by andriusezerskis**
## Purpose / Description
Multiple changes due to wildcards removal commit added noise to the codebase, so I added the commit's hash to the related git-blame file

## Fixes
* Fixes #16747 

## Approach
Removes the noise from all the modified files in the wildcards imports commit

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -433,3 +433,5 @@ d6d6f93f2df24d89e7875756bddcba67059d16b2
 057dff3fbc728b169a0b3cfd3f36c9e25a49477c
 e3078a2eac2dd0f2bde939250b8fcb41a059a03e
 57958d263252e9914b2c25ef68bffedc1abc292e
+
+a6bc50d5fb7ee0a3bad00192044649e9a0e470d3  # removal of wildcards throughout the codebase
```
Comment (lukstbit) [inline .git-blame-ignore-revs:437]: The commit hash is [c63e93a09f86e455ea8c1e9ba13e6ae572550957](https://github.com/ankidroid/Anki-Android/commits/main/)
Also please place the comment above the commit line.

**Review by lukstbit — CHANGES_REQUESTED** [review]

**Review by lukstbit — APPROVED** [review]
Great, thanks!
Also you should squash the two commits.

**Comment by david-allison** [comment]
Links to: 

c63e93a09f86e455ea8c1e9ba13e6ae572550957

**Review by david-allison — APPROVED** [review]
The linked commit is in `main`, perfect!

**Comment by andriusezerskis** [comment]
I squashed the two commits, thanks for your reviews !

**Pull request merged**